pubspec.yaml

```diff
@@ -2,19 +2,18 @@ name: http_mock_adapter
 description: HTTP mock adapter for Dio & Mockito. By simply defining requests
   and corresponding responses through predefined adapters, you will be able to
   mock requests sent via Dio.
-version: 0.1.5
+version: 1.0.0-nullsafety.1
```

Comment:
I think this should change, but not sure to which version as of yet. Though, I personally am thinking on `0.2.0`. I will ask my colleagues for confirmation regarding the version.